core/services/monitoring/client.go

```diff
@@ -0,0 +1,101 @@
+package monitoring
+
+import (
+	"context"
+	"sync/atomic"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/wsrpc"
+)
+
+type Client interface {
+	commontypes.MonitoringEndpoint
```

Comment:
The interface here should use the `TelemetryRequest` as input, not the `commontypes.MonitoringEndpoint` which is just OCRs implementation detail. OCR service should format and send a `TelemetryRequest` via the telemetry client.

That way other system services could also require the telemetry client as a dependency and use it to send their telemetry in the future.